cmd/tapcli/assets.go

```diff
@@ -463,6 +463,11 @@ var sealBatchCommand = cli.Command{
 			Usage: "the asset ID and signature, separated by a " +
 				"colon",
 		},
+		cli.StringSliceFlag{
+			Name: "signed_group_psbt",
+			Usage: "a signed group PSBT for a single asset group " +
```

Comment:
Same UX question here: should we make it more clear that `group_signatures` and `signed_group_psbt` probably shouldn't be used at the same time? Or at least mention that they both do the same thing, just with a different format.

Reply:
I've modified the doc here and on the RPC fields to specify that these options are mutually exclusive. But I've not used `oneof` to avoid a breaking change. I suppose we might make a radical simplification at one point and make many breaking changes at once or V2.